apps/asap-server/src/controllers/research-outputs.ts

```diff
@@ -131,4 +188,12 @@ export interface ResearchOutputController {
   }) => Promise<ListResearchOutputResponse>;
 
   fetchById: (id: string) => Promise<ResearchOutputResponse>;
+  create: (
+    researchOutputRequest: ResearchOutputRequest,
+  ) => Promise<Partial<ResearchOutputResponse>>;
 }
+
+export type ResearchOutputRequest = Pick<
```

Comment:
hate to be picky but i think we should name the controller input data differently - maybe `....InputData` or something, basically i would try to move away from naming everything `request` and `response` - like i said earlier the controllers should not return API specific responses anyway. 

Unless you think we should for now align this with the response (in which case `request` is appropriate) and change later

Reply:
We've rename it. to ResearchOutputInputData

But the previous name was used as it was complementary to the output.

The output type is: ResearchOutputResponse, therefore we thought the input should be a Request i.e. ResearchOutputRequest